routes/applications.test.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
+
+const request = require("supertest");
+
+const db = require("../db");
+const app = require("../app");
+
+const {
+  commonBeforeAll,
+  commonBeforeEach,
+  commonAfterEach,
+  commonAfterAll,
+  u1Token,
+  u2Token,
+} = require("./_testCommon");
+
+beforeAll(commonBeforeAll);
+beforeEach(commonBeforeEach);
+afterEach(commonAfterEach);
+afterAll(commonAfterAll);
+
+// TODO: add an array of ids in _testCommon to minimize the beforeEach functions needed in these tests.
+// Also, change variable names of u1Token to adminToken
+
+/************************************** GET /jobs/:id */
+
+describe("GET /applications/:username", function () {
+
+  test("works for an existing user", async function () {
+    const resp = await request(app)
+      .get(`/applications/u1`)
+      .set("authorization", `Bearer ${u1Token}`);
+    expect(resp.body).toEqual({
+      apps: [
```

Comment:
I believe we send back the companyHandle and companyName as well in the response